test/services/easypost.test.js

```diff
@@ -11,4 +12,43 @@ describe('EasyPost', function () {
       'Missing required parameter: API Key.',
     );
   });
+
+  it('will log the appropriate values when a request and response hooks are provided', async function () {
+    let requestConfig;
+    const requestHook = (response) => (requestConfig = response);
+    let responseConfig;
+    const responseHook = (response) => (responseConfig = response);
+
+    const client = new EasyPost(process.env.EASYPOST_TEST_API_KEY, {
+      requestHook,
+      responseHook,
+    });
+
+    await client.Address.create(Fixture.caAddress1());
+
+    expect(requestConfig).to.be.an('object');
+    expect(requestConfig.method).to.equal(METHODS.POST);
+    expect(requestConfig.path).to.equal('https://api.easypost.com/v2/addresses');
+    expect(requestConfig.requestHeaders).to.be.an('object');
+    expect(requestConfig.requestHeaders['Content-Type']).to.equal('application/json');
+    expect(requestConfig.requestTimestamp).to.be.a('number');
+    expect(requestConfig.requestUUID).to.be.a('string');
+
+    expect(responseConfig).to.be.an('object');
+    expect(responseConfig.method).to.equal(METHODS.POST);
+    expect(responseConfig.path).to.equal('https://api.easypost.com/v2/addresses');
+    expect(responseConfig.requestHeaders).to.be.an('object');
+    expect(responseConfig.requestHeaders['Content-Type']).to.equal('application/json');
+    expect(responseConfig.requestTimestamp).to.be.a('number');
+    expect(responseConfig.requestUUID).to.be.a('string');
+    expect(responseConfig.httpStatus).to.be.a('number');
+    expect(responseConfig.responseBody).to.be.an('object');
+    expect(responseConfig.responseBody.object).to.equal('Address');
+    expect(responseConfig.responseHeaders).to.be.an('object');
+    expect(responseConfig.responseHeaders['content-type']).to.contain('application/json');
+    expect(responseConfig.responseTimestamp).to.be.a('number');
+    expect(responseConfig.responseTimestamp).to.be.greaterThanOrEqual(
+      responseConfig.requestTimestamp,
+    );
+  });
```

Comment:
We need a test for unsubscribe request and response hooks. I believe we are missing unsubscribe functionality

Reply:
Hmmm... yea I guess this is a little different from how it is done in the other diffs. Alrighty. I'll change this API to use `subscribe` and `unsubscribe` methods instead